tests/conftest.py

```diff
@@ -154,14 +165,14 @@ def _generate_calc_job_node(entry_point_name, computer, test_name=None, inputs=N
 
 
 @pytest.fixture(scope='session')
-def generate_upf_data():
+def generate_upf_data(filepath_tests):
     """Return a `UpfData` instance for the given element a file for which should exist in `tests/fixtures/pseudos`."""
 
     def _generate_upf_data(element):
         """Return `UpfData` node."""
         from aiida.orm import UpfData
 
-        filename = os.path.join('tests', 'fixtures', 'pseudos', '{}.upf'.format(element))
+        filename = os.path.join(filepath_tests, 'fixtures', 'pseudos', '{}.upf'.format(element))
         filepath = os.path.abspath(filename)
```

Comment:
This

Reply:
Ah you mean the separate `os.path.abspath` call?

Reply:
Yeah, if ``filepath_tests`` is absolute, there's no need to re-resolve it.